code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -282,12 +282,22 @@
 		if(!LH.target)
 			var/datum/objective/A = new
 			A.owner = user.mind
+			var/list/datum/team/teams = list()
+			for(var/datum/antagonist/antag in user.mind.antag_datums)
+				var/datum/team/T = antag.get_team()
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names